src/modules/launcher/Plugins/Microsoft.Plugin.Calculator/CalculateResult.cs

```diff
@@ -9,13 +9,15 @@ namespace Microsoft.Plugin.Calculator
 {
     public struct CalculateResult : IEquatable<CalculateResult>
     {
+        public bool ValidResult { get; set; }
```

Comment:
[clarification] Can we use `public decimal? Result` and set it to null for incorrect results?